sqlparser/lib/src/ast/visitor.dart

```diff
@@ -122,6 +123,11 @@ class RecursiveVisitor<A, R> implements AstVisitor<A, R> {
     return visitTableInducingStatement(e, arg);
   }
 
+  @override
+  R visitCreateViewStatement(CreateViewStatement e, A arg) {
+    return visitStatement(e, arg);
```

Comment:
Since `CreateViewStatement` is a `SchemaStatement`, let's use the more accurate

```suggestion
    return visitSchemaStatement(e, arg);
```

Reply:
done